Tools/Scripts/webkitpy/tool/commands/download.py

```diff
@@ -333,9 +333,9 @@ def _prepare_state(self, options, args, tool):
                 # We use the earliest revision for the bug info
                 if revision == earliest_revision:
                     state["bug_blocked"] = commit_info.bug_id()
-                    cc_list = sorted([party.bugzilla_email()
+                    cc_list = sorted([party.email
                             for party in commit_info.responsible_parties()
-                            if party.bugzilla_email()])
+                            if getattr(party, 'email', None)])
```

Comment:
Could you explain a bit to me that why we choose `getattr(party, 'email', None)` instead of `hasattr(party, 'email')` or just `party.email` ?

Reply:
Because `getattr` was the first thing that came to mind, should really be `hasattr`

Reply:
Or not, looks like `hasattr` doesn't play nice with `@property`